src/IPinfo/Utilities/CountryHelper.cs

```diff
@@ -12,6 +12,7 @@ namespace IPinfo.Utilities
     internal static class CountryHelper
     {
         private const string ConutriesJsonFilePathName = "IPinfo.Utilities.Countries.json";
+        private const string EUconutriesJsonFilePathName = "IPinfo.Utilities.EUcountries.json";
```

Comment:
Can we rename the file `EuCountries`? Looks weird that `countries` is lowercase like this.